audio.py

```diff
@@ -1,8 +1,11 @@
+import argparse
```

Comment:
Please move `import argparse` and `import time` into the `if __name__ == '__main__'` block: as done in https://github.com/vadimkantorov/convasr/blob/master/metrics.py#L614

Reply:
done